plugins/domains/functions

```diff
@@ -66,15 +66,13 @@ domains_add() {
   verify_app_name "$1"
   local APP="$1"; local APP_VHOST_PATH="$DOKKU_ROOT/$APP/VHOST"
 
-  if [[ $(egrep -w "^${2}$" "$APP_VHOST_PATH" > /dev/null 2>&1; echo $?) -eq 0 ]]; then
-    dokku_log_fail "$2 is already defined for $APP"
-    exit 1
-  fi
-  shift 1
```

Comment:
You'll need to include the `shift 1`. Otherwise, the first arg in `$@` is the app name. This is also why the tests are failing.